ms-windows/mxe/build-mxe.sh

```diff
@@ -24,6 +24,8 @@ set -e
 #        The artifact will be saved as a zip package in the directory
 #        from which this script is launched.
 
+# shellcheck disable=SC2155
```

Comment:
Why not fix this instead? https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
Because it's nonsense, pwd in that case will never return false

Reply:
But I guess I have no choice than wasting my time :(

Reply:
$PWD?